src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -167,8 +167,8 @@ private void testEmail() {
         message = "Email sent, it should arrive shortly, otherwise check your spam folder";
         messageType = JOptionPane.INFORMATION_MESSAGE;
       } catch (final IOException ioe) {
-        message = "Unable to send email; see TripleA Console for details.";
-        ClientLogger.logError(ioe);
+        message = "Unable to send email, " + ioe.getMessage();
```

Comment:
The exception message was specifically removed from the UI because, in some cases, it can be several thousand characters long, which was observed to crash the window manager because the width of the resulting dialog was too large.  In a nutshell, the messages returned from an SMTP server are probably not appropriate for the end user.

Please see #2598 for details.

Reply:
An error message of "unable to perform action" without further details needs some help (I'd say it is not acceptable for the application to just say "no" without any mention of how to fix or what went wrong). Do you have any objection to truncating the error message? I also added a note to check SMPT credentials: 2298cfd1f

Reply:
No objection to truncating.  I actually considered truncating or wrapping the text (see [here](https://github.com/triplea-game/triplea/issues/2595#issuecomment-343696573)).  Don't remember right now why I abandoned that idea.